scripts/install

```diff
@@ -52,10 +52,28 @@ function strimzi {
 }
 
 function hindsight {
+    local -r location="${3:?Location required.}"
+
+    if [[ "$location" == "aws" ]]; then
+        aws_deploy "${@}"
+    else
+        local_deploy "${@}"
+    fi
+}
+
+
+function aws_deploy {
     local -r release="${1:?Release name required.}"
     local -r ns="${2:?Namespace required.}"
     local -r location="${3:?Location required.}"
-    shift 3
+    local -r environment="${4:?Environment required.}"
+    shift 4
+
+    local -r username=$(aws secretsmanager get-secret-value --secret MetastoreInfo-${environment} | jq -rc .SecretString | jq .username | tr -d "\n\"")
+    local -r password=$(aws secretsmanager get-secret-value --secret MetastoreInfo-${environment} | jq -rc .SecretString | jq .password | tr -d "\n\"")
+    local -r postgresEndpointAddress=$(aws secretsmanager get-secret-value --secret MetastoreInfo-${environment} | jq -rc .SecretString | jq .endpoint | tr -d "\n\"")
+    local -r bucketName=$(aws secretsmanager get-secret-value --secret MetastoreInfo-${environment} | jq -rc .SecretString | jq .bucket | tr -d "\n\"")
+    local -r redisEndpointAddress=$(aws secretsmanager get-secret-value --secret RedisInfo-${environment} | jq -rc .SecretString | jq .endpoint | tr -d "\n\"")
```

Comment:
I'm okay with this for the moment but I don't think long-term we want to couple ourselves to any specific implementations of infrastructure deployments, especially in the generic project repo. I think this makes _slightly_ more sense in the inhindsight/hindsight-aws repo but i think it would be preferable to assume a standard set of flags that get passed to our script or have the script take a config file and it's an exercise left to the reader how they generate those flags or that config file

@jdenen thoughts?

Reply:
This is probably fair, although making things and exercise for the reader means they're an exercise for us and I want this thing to be easily deployed without having to jump through hoops.  I am cool with these implementation details /for us/ living somewhere else though and would be behind hindsight proper not leaking those.  Yesterday johnson was cool with not making that move yet but id be interested to see if he changes his mind, im ambivelent in the short term.

Reply:
Punt on it for now.

This script isn't required for using Hindsight, so I don't see it as a big deal to tie ourselves to AWS directly here. If you're deploying to Azure/GCP/Digitial Ocean, don't use the script.

Reply:
Long term, I agree that we'll want to move away from this. But JSR is low for that line of work at the moment.